modules/components/forms/text_input.cjsx

```diff
@@ -21,6 +21,7 @@ module.exports = class TextInput extends RestFormElement
 
   @defaultProps =
     inputType: 'text'
+    value:     ''
```

Comment:
In addition, it'll mean all devs will have to add `_.extend {}, RestFormElement.defaultProps, ...` in their projects.